pwndbg/gdblib/functions.py

```diff
@@ -67,3 +68,17 @@ def base(name_pattern: gdb.Value | str) -> int:
         if name in p.objfile:
             return p.vaddr
     raise ValueError(f"No mapping named {name}")
+
+
+@GdbFunction(only_when_running=True)
+def addressify(hex_string: gdb.Value | str) -> int:
```

Comment:
```suggestion
def hex2ptr(hex_string: gdb.Value | str) -> int:
```

How is `gdb.Value` accepted here? what does it do?

Reply:
II think when you call `p $hex2ptr("00 70 ...")`  GDB treats "00 70..." as GDB expression rather than a raw string, so expression gets evaluated to `gdb.Value` object